### ISSUE TITLE
Kahlan does not like backslashed global functions

### ISSUE THREAD

**Issue by SmoshySmosh** [kahlan/kahlan#301]
In my project, I have a dependency of [Symfony Filesystem](https://github.com/symfony/filesystem), which puts backslashes on all of the global functions to avoid a namespace lookup. Kahlan does not seem to be liking this at all.

To test the issue, I went into my composer folder and setup a var_dump() on the problem code. Upon running Kahlan, I found the behavior to be a little wonky, but I do not see the same thing when running the code with just straight PHP.

For example here is the [`box`](https://github.com/symfony/filesystem/blob/master/Filesystem.php#L742-L757) function inside of the Filesystem library. I have added a few var_dump() lines to debug the issue.
```php
private static function box($func)
{
    self::$lastError = null;
    \set_error_handler(__CLASS__.'::handleError');
    try {
        var_dump('Fails:', \func_get_args());
        var_dump('Works:', func_get_args());
        $result = \call_user_func_array($func, \array_slice(\func_get_args(), 1));
        \restore_error_handler();

        return $result;
    } catch (\Throwable $e) {
    } catch (\Exception $e) {
    }
    \restore_error_handler();

    throw $e;
}
```

The output is as follows:
```bash
./vendor/symfony/filesystem/Filesystem.php:747:
string(6) "Fails:"

./vendor/symfony/filesystem/Filesystem.php:747:
bool(false)

./vendor/symfony/filesystem/Filesystem.php:748:
string(6) "Works:"

./vendor/symfony/filesystem/Filesystem.php:748:
array(4) {
  [0] =>
  string(5) "mkdir"
  [1] =>
  string(85) "/var/folders/wg/44g3b1b96z3398_9mtv9yl100000gp/T/81e05913-5611-40a2-abf4-a58e0cf220f8"
  [2] =>
  int(511)
  [3] =>
  bool(true)
}
```

I'm not exactly sure why this is happening, but I am currently blocked by it.

**Comment by jails** [comment]
Hi !
Indeed there was an issue on backslashed introspection functions like `func_get_args`.
I just fixed it in the dev branch https://github.com/kahlan/kahlan/commit/ab436a4a2f6a55c944e3f389a83625e780586f1d
So feel free to test it so I can create a tag.

Note: don't forget to clear your cache after the upgrade (i.e. `kahlan --cc`)

**Comment by SmoshySmosh** [comment]
That was fast! Thank you very much. I will test it in an hour or so and get back to you.

**Comment by SmoshySmosh** [comment]
Update: It seems to be working now. You may go ahead and cut a new release. 👍

**Comment by SmoshySmosh** [comment]
@jails Checking back in on this issue, would love to see it released so I can move away from using the commit hash in my composer.json. 😄

**Comment by jails** [comment]
Fixed in 4.1.4
